rhobs/alerting/data_plane/prometheus.release_service_alerts.yaml

```diff
@@ -83,3 +83,18 @@ spec:
         alert_team_handle: <!subteam^S03SVBS426R>
         runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/release-service/release-controller-availability.md?ref_type=heads
         team: release
+
+    - alert: ReleaseServiceControllerManagerPodNotReady
+      expr: |
+        sum by (source_cluster, namespace) (kube_pod_container_status_ready{container="manager",namespace="release-service",source_cluster=~"stone-prd-rh01|stone-prod-p02"}) == 0
+      for: 10m
+      labels:
+        severity: high
+      annotations:
+        summary: >-
+          The Release Service Controller Manager is down in the {{ $labels.source_cluster }} for the last 10 minutes
+        description: >-
+          The Release Service Controller Manager is down in the {{ $labels.source_cluster }} for the last 10 minutes
+        alert_team_key: "{{ $labels.namespace }}"
```

Comment:
You should actually use `alert_routing_key` instead of `alert_team_key`

Reply:
ops. route, not team :D.